main.go

```diff
@@ -173,8 +175,12 @@ func parseFlags() *config {
 	cfg.promlogConfig.Format = &promlog.AllowedFormat{}
 	a.Flag("log.format", "Output format of log messages. One of: [logfmt, json]").
 		Envar("PROMBQ_LOG_FORMAT").Default("logfmt").SetValue(cfg.promlogConfig.Format)
-
+	googleProjectIDFlagCause := a.Flag("googleProjectID", "The GCP Project ID is mandatory when googleAPIjsonkeypath is not provided").Envar("PROMBQ_GCP_PROJECT_ID")
```

Comment:
Can you please move this line under the one for `googleAPIjsonkeypath`?

Reply:
Hi thanks for the quick feedback
 the first parse() gets the err 'Type is missing' ... because type was not applied to the (kingpin) cause.
And the subsequent if statement resulted in execution due to that faillure
fixed by adding 'Type' to the 'cause' prior to executing the first parse.